login/routers/controllers/UserExistCheck.js

```diff
@@ -4,19 +4,19 @@ const SequelizeOperators = Sequelize.Op;
 const UserModel = require('../../db/models/UserModel.js');
 const createUserAccount = require('../../db/CreateUserAccount.js');
 
-function CopyCheck(registrationRequest, response) {
+function UserExist(registrationRequest, response) {
```

Comment:
This function is either creating a new user or sending a 400 if a user already exists, the name doesn't reflect that.

Reply:
i read about 409,and briefly it is "conflict" but when i open wide-ranging description of it,there was about put requests